chef/cookbooks/tempest/recipes/config.rb

```diff
@@ -337,9 +340,50 @@
   cinder_backend2_name = backend_names[1]
 end
 
-compute_nodes = search(:node, "roles:nova-multi-compute-kvm") || []
-use_resize = compute_nodes.length > 1
-use_livemigration = nova[:nova][:use_migration] && compute_nodes.length > 1
+kvm_compute_nodes = search(:node, "roles:nova-multi-compute-kvm") || []
+docker_compute_nodes = search(:node, "roles:nova-multi-compute-docker") || []
+
+use_resize = kvm_compute_nodes.length > 1
+use_livemigration = nova[:nova][:use_migration] && kvm_compute_nodes.length > 1
+
+if !docker_compute_nodes.empty? && kvm_compute_nodes.empty?
+  image_name = "cirros"
+
+  bash "find tempest test image for docker" do
+    code <<-EOH
+DOCKER_IMAGE_ID=$(glance #{insecure} image-list \
+    --name #{image_name} \
+    --container-format docker \
+    --is-public True \
+    --page-size 1 \
+    2> /dev/null | tail -n 2 | head -n 1 | awk '{ print $2 }')
+[ -n "$DOCKER_IMAGE_ID" ] && echo "$DOCKER_IMAGE_ID" > #{docker_image_id_file}
```

Comment:
looks like we should include https://github.com/crowbar/barclamp-tempest/pull/159 as welll